### PR TITLE
Add descriptions to all TS units

### PR DESCRIPTION
**Pull request by abcdefg30**
Descriptions "inspired" by the articles found at
http://cnc.wikia.com/wiki/Category:Tiberian_Sun_Nod_Arsenal and
http://cnc.wikia.com/wiki/Category:Tiberian_Sun_GDI_Arsenal .

Suggestions for improvement wanted. :)

### CODE REVIEW

File: mods/ts/rules/aircraft.yaml [hunk 1]
```diff
@@ -171,10 +172,11 @@ TRNSPORT:
 		Cost: 750
 	Tooltip:
 		Name: Carryall
+		Description: Vehicle Transport Helicopter.\n  Unarmed
```
Comment (penev92) [inline mods/ts/rules/aircraft.yaml]: I wouldn't say "Helicopter", even though it's VTOL. Just say "aircraft" ?

File: mods/ts/rules/gdi-infantry.yaml [hunk 2]
```diff
@@ -91,12 +92,12 @@ GHOST:
 		Cost: 1750
 	Tooltip:
 		Name: Ghost Stalker
-		Description: Elite commando infantry, armed with \ndual pistols and C4.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles\n  Special Ability: Destroy Building with C4\n\nMaximum 1 can be trained
-		BuildLimit: 1
+		Description: Elite commando infantry, armed with \ndual pistols and C4.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles\n  Special Ability: Destroy Building with C4\nMaximum 1 can be trained
```
Comment (penev92) [inline mods/ts/rules/gdi-infantry.yaml]: You should fix the Tanya description here to be actually valid for the Ghost stalker, who doesn't have dual pistols.

File: mods/ts/rules/gdi-structures.yaml [hunk 3]
```diff
@@ -107,7 +107,7 @@ GAWEAP:
 		Cost: 2000
 	Tooltip:
 		Name: GDI War Factory
-		Description: Assembly point for\nvehicle reinforcements
+		Description: Assembly point for\nvehicle reinforcements.
```
Comment (penev92) [inline mods/ts/rules/gdi-structures.yaml]: This sounds like it was copied from the TD Nod Airstrip.

File: mods/ts/rules/gdi-structures.yaml [hunk 4]
```diff
@@ -272,14 +272,14 @@ GATECH:
 		Cost: 2000
 	Tooltip:
 		Name: GDI Tech Center
-		Description: Required for high-\ntech research
+		Description: Required for high-tech research.
```
Comment (penev92) [inline mods/ts/rules/gdi-structures.yaml]: How about "Provides access to advanced GDI technologies."

File: mods/ts/rules/gdi-support.yaml [hunk 5]
```diff
@@ -134,7 +134,7 @@ GAVULC:
 		Cost: 150
 	Tooltip:
 		Name: Vulcan Tower
-		Description: Basic base defense. \nRequires no power to operate.\n  Strong vs infantry and light armor\n  Cannot target Aircraft
+		Description: Basic base defense.\nRequires no power to operate.\n  Strong vs infantry and light armor\n  Cannot target Aircraft
```
Comment (penev92) [inline mods/ts/rules/gdi-support.yaml]: I don't think the "no power" part is accurate.

File: mods/ts/rules/gdi-vehicles.yaml [hunk 6]
```diff
@@ -4,7 +4,7 @@ APC:
 		Cost: 600
 	Tooltip:
 		Name: Amphibious APC
-		Description: Armored infantry transport and mobile AA\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
+		Description: Armored infantry transport and mobile AA.\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/gdi-vehicles.yaml]: Actually unarmed.

File: mods/ts/rules/shared-vehicles.yaml [hunk 7]
```diff
@@ -120,6 +121,7 @@ GGHUNT:
 		Cost: 1000
 	Tooltip:
 		Name: Hunter-Seeker Droid
+		Description: Suicide hovercraft-driod.
```
Comment (pchote) [inline mods/ts/rules/shared-vehicles.yaml]: This isn't buildable, so doesn't need a description.

File: mods/ts/rules/nod-structures.yaml [hunk 8]
```diff
@@ -197,7 +197,7 @@ NARADR:
 		Cost: 1000
 	Tooltip:
 		Name: Nod Radar
-		Description: Provides radar screen
+		Description: Provides radar screen.
```
Comment (pchote) [inline mods/ts/rules/nod-structures.yaml]: I think we can come up with a better description than this.

File: mods/ts/rules/nod-structures.yaml [hunk 9]
```diff
@@ -236,7 +236,7 @@ NATECH:
 		Cost: 2000
 	Tooltip:
 		Name: Nod Tech Center
-		Description: Required for high-\ntech research
+		Description: Required for high-tech research.
```
Comment (pchote) [inline mods/ts/rules/nod-structures.yaml]: Likewise here.

File: mods/ts/rules/gdi-vehicles.yaml [hunk 10]
```diff
@@ -145,10 +145,12 @@ HMEC:
 		Cost: 3000
 	Tooltip:
 		Name: Mammoth Mk. II
+		Description: Slow heavy walker.\n  Strong vs Infantry, Tanks and Aircraft\n  Weak vs Nothing\nMaximum one can be build.
```
Comment (penev92) [inline mods/ts/rules/gdi-vehicles.yaml]: + "at a time".
Also this uses "one" and the Ghost stalker says "1".

File: mods/ts/rules/gdi-vehicles.yaml [hunk 11]
```diff
@@ -178,6 +181,7 @@ SONIC:
 		Cost: 1300
 	Tooltip:
 		Name: Disruptor
+		Description: Advanced anti-armour vehicle.\n  Strong vs Infantry, Tanks\n  Weak vs Aircraft
```
Comment (penev92) [inline mods/ts/rules/gdi-vehicles.yaml]: [23:57] <penev> isn't the disruptor more like an anti-defence tank?
[23:57] <penev> iirc it's great against buildings, has a long range, and dies like a fly to both infantry and vehicles

File: mods/ts/rules/nod-structures.yaml [hunk 12]
```diff
@@ -114,7 +114,7 @@ NAWEAP:
 		Cost: 2000
 	Tooltip:
 		Name: Nod War Factory
-		Description: Assembly point for\nvehicle reinforcements
```
Comment (penev92) [inline mods/ts/rules/nod-structures.yaml]: This sounds like it was copied from the TD Nod Airstrip.

File: mods/ts/rules/nod-structures.yaml [hunk 13]
```diff
@@ -43,7 +43,7 @@ NAAPWR:
 		Cost: 600
 	Tooltip:
 		Name: Advanced Power Plant
-		Description: Provides more power for structures
+		Description: Provides more power for structures.
```
Comment (penev92) [inline mods/ts/rules/nod-structures.yaml]: I'm not convinced about this one.

File: mods/ts/rules/nod-structures.yaml (hunk 9 above)
Comment (penev92) [inline mods/ts/rules/nod-structures.yaml]: I suggested an alternative for the GDI tech-center, which can be used here as well.

File: mods/ts/rules/nod-structures.yaml [hunk 14]
```diff
@@ -114,7 +114,7 @@ NAWEAP:
 		Cost: 2000
 	Tooltip:
 		Name: Nod War Factory
-		Description: Assembly point for\nvehicle reinforcements
+		Description: Assembly point for\nvehicle reinforcements.
```
Comment (penev92) [inline mods/ts/rules/nod-structures.yaml]: This sounds like it was copied from the TD Nod Airstrip.

File: mods/ts/rules/nod-structures.yaml [hunk 15]
```diff
@@ -261,7 +261,7 @@ NASTLH:
 		Cost: 2500
 	Tooltip:
 		Name: Stealth Generator
-		Description: Generates a cloaking field
+		Description: Generates a cloaking field.
```
Comment (penev92) [inline mods/ts/rules/nod-structures.yaml]: + "in a wide area around it to hide your forces from the enemy"
or something of the sort.

File: mods/ts/rules/nod-support.yaml [hunk 16]
```diff
@@ -113,7 +113,7 @@ NASAM:
 		Cost: 600
 	Tooltip:
 		Name: S.A.M. Site
-		Description: Nod Anti-Air base defense. \nRequires power to operate.\n  Strong vs all Aircraft\n  Cannot target ground units
+		Description: Nod Anti-Air base defense.\nRequires power to operate.\n  Strong vs all Aircraft\n  Cannot target ground units
```
Comment (penev92) [inline mods/ts/rules/nod-support.yaml]: "Aircraft" in all 3 Nod defenses is needlessly capitalized IMO.
Also no need for "all" here.

File: mods/ts/rules/nod-vehicles.yaml [hunk 17]
```diff
@@ -199,7 +201,7 @@ SAPC:
 		Cost: 800
 	Tooltip:
 		Name: Subterranean APC
-		Description: Armored infantry transport and mobile AA\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
+		Description: Armored infantry transport and mobile AA.\nIs able to move underground.\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: Strong versus nothing, actually.
And I think it hardly qualifies as "armored".
How about "Troop transport that can move underground to avoid detection." ?

File: mods/ts/rules/nod-vehicles.yaml [hunk 18]
```diff
@@ -256,7 +258,7 @@ STNK:
 		Cost: 1100
 	Tooltip:
 		Name: Stealth Tank
-		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry.
+		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: I don't recall it being very long-ranged :?
Also "Can be spotted by infantry at close range".
And "Strong vs Vehicles, Tanks, Aircraft" has one ground-based group of units too much :P

File: mods/ts/rules/shared-support.yaml [hunk 19]
```diff
@@ -29,7 +29,7 @@ NAPULS:
 		Cost: 1000
 	Tooltip:
 		Name: EMP Cannon
-		Description: Disables vehicles. \nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
+		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
```
Comment (penev92) [inline mods/ts/rules/shared-support.yaml]: Also doesn't do anything to infantry, so maybe we should mention that?

File: mods/ts/rules/shared-vehicles.yaml (hunk 7 above)
Comment (abcdefg30) [inline mods/ts/rules/shared-vehicles.yaml]: According to cnc wikia it is.

File: mods/ts/rules/gdi-vehicles.yaml (hunk 10 above)
Comment (abcdefg30) [inline mods/ts/rules/gdi-vehicles.yaml]: Ops, seems like git ate the `one` -> `1` change.
> Add "at a time" please

We do it like this everywhere else, so:
a) change it here and at several other places
b) let it as is
?

Comment (penev92) [inline mods/ts/rules/gdi-vehicles.yaml]: blah...
It sounds dumb without it, but if we do it everywhere it's your call.

File: mods/ts/rules/nod-support.yaml (hunk 16 above)
Comment (abcdefg30) [inline mods/ts/rules/nod-support.yaml]: Actually it should be capitalized everywhere.

**Comment by abcdefg30** [comment]
Updated.

File: mods/ts/rules/nod-vehicles.yaml [hunk 20]
```diff
@@ -256,7 +258,7 @@ STNK:
 		Cost: 1100
 	Tooltip:
 		Name: Stealth Tank
-		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry.
+		Description: Weak armoured tank that can cloak.\nCan be spotted by infantry at close range.\n  Strong vs Tanks, Aircraft\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: Lightly armored tank with a personal stealth generator.\nArmed with missiles.\nCan be spotted by infantry at close range.

**Comment by abcdefg30** [comment]
Updated.

But I think we need a decision on how to handle `Strong/Weak vs ...` and `Cannot target ...`.
If actor `A` can't target e.g. `Aircraft`, what of the following should the description say?
a) Nothing about that
b) `Cannot target Aircraft`
c) `Weak vs Aircraft`

File: mods/ts/rules/nod-vehicles.yaml [hunk 21]
```diff
@@ -256,7 +258,7 @@ STNK:
 		Cost: 1100
 	Tooltip:
 		Name: Stealth Tank
-		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry.
+		Description: Light armoured tank equipped with a personal\nstealth generator. Armed with missiles.\nCan be spotted by infantry at close range.\n  Strong vs Vehicles, Aircraft\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: **Lightly*** armoured.

File: mods/ts/rules/shared-support.yaml [hunk 22]
```diff
@@ -29,7 +29,7 @@ NAPULS:
 		Cost: 1000
 	Tooltip:
 		Name: EMP Cannon
-		Description: Disables vehicles. \nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
+		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs Tanks\n  Cannot target Aircraft
```
Comment (penev92) [inline mods/ts/rules/shared-support.yaml]: In TS "tanks" is pretty iffy, because there are only like 3 "tanks" total. I'd rather go with "vehicles".

File: mods/ts/rules/aircraft.yaml [hunk 23]
```diff
@@ -66,7 +66,7 @@ ORCA:
 		Cost: 1000
 	Tooltip:
 		Name: Orca Fighter
-		Description: Helicopter Gunship with Missiles.\n  Strong vs Buildings, Tanks\n  Weak vs Infantry
+		Description: Helicopter gunship with dual missile launchers.\n  Strong vs Buildings, Vehicles\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/aircraft.yaml]: How about "Fast assault gunship armed with dual missile launchers" ?
I don't like the use of "helicopter", because they aren't, so I did some reading [here](http://cnc.wikia.com/wiki/Orca_Fighter), [here](http://cnc.wikia.com/wiki/Orca_aircraft) and [here](http://gaming.wikia.com/wiki/Units_%28Command_%26_Conquer:_Tiberian_Sun%29).

File: mods/ts/rules/aircraft.yaml [hunk 24]
```diff
@@ -104,6 +104,7 @@ ORCAB:
 		Cost: 1600
 	Tooltip:
 		Name: Orca Bomber
+		Description: Heavy Bomber.\n  Strong vs Buildings, Vehicles\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ts/rules/aircraft.yaml]: I think "bomber" wants to be lowercase.

File: mods/ts/rules/aircraft.yaml [hunk 25]
```diff
@@ -236,6 +239,7 @@ APACHE:
 		Cost: 1000
 	Tooltip:
 		Name: Harpy
+		Description: Support gunship.\n  Strong vs Infantry, Light Vehicles, Aircraft\n  Weak vs Buildings, Tanks
```
Comment (penev92) [inline mods/ts/rules/aircraft.yaml]: How about "Anti-personnel support gunship armed with dual chain guns."
I admit, I read stuff [here](http://cnc.wikia.com/wiki/Harpy) :D

File: mods/ts/rules/gdi-infantry.yaml [hunk 26]
```diff
@@ -8,7 +8,7 @@ E2:
 		Cost: 200
 	Tooltip:
 		Name: Disc Thrower
-		Description: Infantry armed with grenades. \n  Strong vs Buildings, Infantry\n  Weak vs Vehicles
+		Description: Infantry armed with grenades.\n  Strong vs Buildings, Infantry\n  Weak vs Vehicles
```
Comment (penev92) [inline mods/ts/rules/gdi-infantry.yaml]: "with special explosive discs"
Also strong vs vehicles, according to http://cnc.wikia.com/wiki/Disk_Thrower

File: mods/ts/rules/gdi-vehicles.yaml [hunk 27]
```diff
@@ -4,7 +4,7 @@ APC:
 		Cost: 600
 	Tooltip:
 		Name: Amphibious APC
-		Description: Armored infantry transport and mobile AA\n  Strong vs Aircraft, Vehicles\n  Weak vs Infantry
+		Description: Armored infantry transport.\n  Unarmed
```
Comment (penev92) [inline mods/ts/rules/gdi-vehicles.yaml]: "Can move on water".

File: mods/ts/rules/gdi-vehicles.yaml [hunk 28]
```diff
@@ -145,6 +146,7 @@ HMEC:
 		Cost: 3000
 	Tooltip:
 		Name: Mammoth Mk. II
+		Description: Slow heavy walker.\n  Strong vs Infantry, Vehicles, Aircraft and Buildings\n  Weak vs Nothing\nMaximum 1 can be build.
```
Comment (penev92) [inline mods/ts/rules/gdi-vehicles.yaml]: "armed with dual railguns and rocket launchers".

Also you can add something like "GDI's ultimate war machine", if you can make it sound good ;)

File: mods/ts/rules/nod-infantry.yaml [hunk 29]
```diff
@@ -32,7 +32,7 @@ CYBORG:
 		Cost: 650
 	Tooltip:
 		Name: Cyborg Infantry
-		Description: Cybernetic infantry unit.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles
+		Description: Cybernetic infantry unit.\n  Strong vs Infantry, Buildings\n  Weak vs Vehicles, Aircraft
```
Comment (penev92) [inline mods/ts/rules/nod-infantry.yaml]: They were pretty good against vehicles as far as I recall. At least against light armor.

File: mods/ts/rules/nod-vehicles.yaml [hunk 30]
```diff
@@ -4,7 +4,7 @@ BGGY:
 		Cost: 500
 	Tooltip:
 		Name: Attack Buggy
-		Description: Fast scout & anti-infantry vehicle.\n  Strong vs Infantry, Vehicles\n  Weak vs Tanks, Aircraft
+		Description: Fast scout and anti-infantry vehicle.\n  Strong vs Infantry, Vehicles\n  Weak vs Tanks, Aircraft
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: IMO switch "Vehicles" for "light armor" and "Tanks" for "Vehicles".

File: mods/ts/rules/nod-vehicles.yaml [hunk 31]
```diff
@@ -73,6 +73,7 @@ TTNK:
 		Cost: 800
 	Tooltip:
 		Name: Tick Tank
+		Description: Main battle tank.\nCan deploy to gain extra protection.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: "Nod's main battle tank."

File: mods/ts/rules/nod-vehicles.yaml [hunk 32]
```diff
@@ -118,6 +119,7 @@ ART2:
 		Cost: 975
 	Tooltip:
 		Name: Artillery
+		Description: Mobile Artillery. Needs\nto deploy in order to shoot.\n  Strong vs ground units\n  Cannot target Aircraft
```
Comment (penev92) [inline mods/ts/rules/nod-vehicles.yaml]: Will this fit in the tooltip if you moved "Needs" to the second line, so it's a sentence per line?

**Comment by penev92** [comment]
For what it's worth, I am actually sorry about the amount of bike-shedding done here.

File: mods/ts/rules/aircraft.yaml (hunk 24 above)
Comment (pchote) [inline mods/ts/rules/aircraft.yaml]: I agree.

File: mods/ts/rules/aircraft.yaml (hunk 25 above)
Comment (pchote) [inline mods/ts/rules/aircraft.yaml]: Tanks &rarr; Vehicles?  TS only has a couple of tanks, and I don't think any of them are AA?

File: mods/ts/rules/gdi-structures.yaml [hunk 33]
```diff
@@ -233,7 +233,7 @@ GARADR:
 		Cost: 1000
 	Tooltip:
 		Name: GDI Radar
-		Description: Provides radar screen
+		Description: Provides an overview of the battlefield.\nCan detect cloaked units.\nRequires power to operate.
```
Comment (pchote) [inline mods/ts/rules/gdi-structures.yaml]: Could it detect cloak in the original?

File: mods/ts/rules/gdi-support.yaml [hunk 34]
```diff
@@ -166,7 +166,7 @@ GACSAM:
 		Cost: 300
 	Tooltip:
 		Name: SAM. Upgrade
-		Description: GDI Anti-Air base defense. \nRequires power to operate.\n  Strong vs all Aircraft\n  Cannot target ground units
+		Description: GDI Anti-Air base defense.\nRequires power to operate.\n  Strong vs Aircraft\n  Cannot target ground units
```
Comment (pchote) [inline mods/ts/rules/gdi-support.yaml]: None of our component tower upgrades require power.

File: mods/ts/rules/gdi-vehicles.yaml [hunk 35]
```diff
@@ -34,7 +34,7 @@ HVR:
 		Cost: 900
 	Tooltip:
 		Name: Hover MLRS
-		Description: Hover Multi-Launch Rocket System
+		Description: Hover Multi-Launch Rocket System.\n  Strong vs Vehicles, Aircraft\n  Weak vs Infantry
```
Comment (pchote) [inline mods/ts/rules/gdi-vehicles.yaml]: This doesn't describe the unit like all the other descriptions.

File: mods/ts/rules/gdi-vehicles.yaml [hunk 36]
```diff
@@ -178,6 +180,7 @@ SONIC:
 		Cost: 1300
 	Tooltip:
 		Name: Disruptor
+		Description: Armoured hightech vehicle with\nlong range and sonic armament.\n  Strong vs Infantry, Vehicles, Buildings\n  Weak vs Aircraft
```
Comment (pchote) [inline mods/ts/rules/gdi-vehicles.yaml]: high-tech

File: mods/ts/rules/shared-support.yaml (hunk 22 above)
Comment (pchote) [inline mods/ts/rules/shared-support.yaml]: I agree.

File: mods/ts/rules/gdi-vehicles.yaml (hunk 28 above)
Comment (pchote) [inline mods/ts/rules/gdi-vehicles.yaml]: build &rarr; built.

File: mods/ts/rules/gdi-vehicles.yaml (hunk 36 above)
Comment (chrisforbes) [inline mods/ts/rules/gdi-vehicles.yaml]: Armored -- convention is to use US English.

File: mods/ts/rules/aircraft.yaml (hunk 25 above)
Comment (penev92) [inline mods/ts/rules/aircraft.yaml]: In that case "Light vehicles" -> "Light armor" would probably be in order.

File: mods/ts/rules/gdi-structures.yaml (hunk 33 above)
Comment (penev92) [inline mods/ts/rules/gdi-structures.yaml]: According to @reaperrr it could, but only at melee range. (like infantry)

File: mods/ts/rules/gdi-support.yaml (hunk 34 above)
Comment (penev92) [inline mods/ts/rules/gdi-support.yaml]: Well YAML says otherwise.

Comment (pchote) [inline mods/ts/rules/gdi-support.yaml]: There is a difference between drawing power and requiring power (becoming disabled when the player is in a low power state).

**Comment by abcdefg30** [comment]
> But I think we need a decision on how to handle `Strong/Weak vs ...` and `Cannot target ...`.
If actor `A` can't target e.g. `Aircraft`, what of the following should the description say?
a) Nothing about that
b) `Cannot target Aircraft`
c) `Weak vs Aircraft`

For the record: We agreed to use `c)` ([IRC logs](http://logs.ihptru.net/?year=2015&month=04&day=20#15:36:06))

**Comment by abcdefg30** [comment]
Updated again.

**Comment by pchote** [comment]
:+1:

File: mods/ts/rules/nod-vehicles.yaml [hunk 37]
```diff
@@ -36,7 +36,7 @@ BIKE:
 		Cost: 600
 	Tooltip:
 		Name: Attack Cycle
-		Description: Fast scout vehicle, armed with \nrockets.\n  Strong vs Vehicles, Aircraft\n  Weak vs Infantry
+		Description: Fast scout vehicle, armed with\nrockets.\n  Strong vs Vehicles, Aircraft\n  Weak vs Infantry
```
Comment (reaperrr) [inline mods/ts/rules/nod-vehicles.yaml]: This is wrong. The Bike's missiles are currently anti-ground only (same as in the original), so it should be just 'Strong vs Vehicles'.

File: mods/ts/rules/shared-support.yaml [hunk 38]
```diff
@@ -28,7 +28,7 @@ NAPULS:
 		Cost: 1000
 	Tooltip:
 		Name: EMP Cannon
-		Description: Disables vehicles. \nRequires power to operate.\n  Strong vs all ground units\n  Cannot target Aircraft
+		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
```
Comment (reaperrr) [inline mods/ts/rules/shared-support.yaml]: In the original it worked vs Cyborgs and Buildings, too, but it currently doesn't in OpenRA, so this description is ok for now.

**Comment by reaperrr** [comment]
:+1: once the wrong Bike description has been fixed.

File: mods/ts/rules/shared-support.yaml (hunk 38 above)
Comment (pchote) [inline mods/ts/rules/shared-support.yaml]: We should fix that.

File: mods/ts/rules/shared-support.yaml (hunk 19 above)
Comment (reaperrr) [inline mods/ts/rules/shared-support.yaml]: Yes, but the EMP needs some tweaking in general (firing offsets, weapon range etc.), so I'd leave that for its own PR.

**Comment by abcdefg30** [comment]
Updated.
